Dockerfile

```diff
@@ -1,8 +1,11 @@
 # Base image
-ARG LITELLM_BASE_IMAGE=python:3.9-slim
+ARG LITELLM_BASE_IMAGE=python:3.9
```

Comment:
As we are separating build and runtime images, can you rename this to `LITELLM_BUILD_IMAGE` to be more explicit?